src/components/historicalTrendChart/historicalTrendChart.tsx

```diff
@@ -267,11 +267,15 @@ class HistoricalTrendChart extends React.Component<
               datum.cost.charts.map((chart, index) => {
                 return this.getChart(chart, index);
               })}
-            <VictoryAxis label={xAxisLabel} style={chartStyles.axis} />
+            <VictoryAxis
+              label={xAxisLabel}
+              style={chartStyles.xAxis}
+              tickValues={[1, 15, 31]}
```

Comment:
Does this not show `31` if the month doesn't have 31 days? Is the intent to show begin, middle, end, of which I feel like you could use the max of the two months (if you have two months of non-zero data).